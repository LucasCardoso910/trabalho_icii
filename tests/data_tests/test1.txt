8
08
12
19
43
45
56
67
95