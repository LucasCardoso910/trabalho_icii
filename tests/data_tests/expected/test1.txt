bubble_sort:28,0
heap_sort:36,58
insert_binary_sort:13,14
insert_sort:7,21
merge_sort:12,32
quick_sort:21,16
selection_sort:28,21
shake_sort:7,0
